runtime/common/tests/xcm_mock/mod.rs

```diff
@@ -96,6 +102,13 @@ pub fn para_ext(para_id: u32) -> sp_io::TestExternalities {
 	)
 	.unwrap();
 
+	pallet_asset_manager::GenesisConfig::<Runtime> {
+		start_id: <ParachainAssetConfig as AssetConfig<Runtime>>::StartNonNativeAssetId::get(),
+		_marker: PhantomData::<Runtime>::default(),
+	}
+	.assimilate_storage(&mut t)
+	.unwrap();
```

Comment:
This is kind-of hidden detail of the tests but it's helpful because it sets the native asset and moves the next asset id along, so the `AssetConfig` impl can be used directly in the tests.